pkg/debug/scripts/cgv2-audit.sh

```diff
@@ -0,0 +1,198 @@
+#!/bin/sh
+# cgv2-audit.sh — inspect cgroup v2 on EVE (run from debug container)
+# - Prefers host view at /hostfs; falls back to container /sys if missing
+# - Dumps controllers, subtree_control, cpuset init, common limits, and PIDs
+# - Optionally installs bpftool via apk when run with: CGV2_INSTALL_BPFTOOL=1 ./cgv2-audit.sh
+#
+# Env knobs:
+#   CGV2_ROOT=/hostfs           # host root prefix (default)
+#   CGV2_INSTALL_BPFTOOL=1      # try to apk add bpftool if missing
+#   CGV2_ALPINE_VER=3.17        # repo version used for bpftool install
+
+set -eu
+
+CGV2_INSTALL_BPFTOOL=1
+CGV2_ALPINE_VER=3.17
+
+ROOT="${CGV2_ROOT:-/hostfs}"
```

Comment:
Is this just to allow someone to override manually? So they can call `CGV2_ROOT=/somewhere/else cgv2-audit.sh`?